Offseason2022/src/main/java/frc/robot/commands/DriveMotorTest.java

```diff
@@ -24,7 +31,13 @@ public void initialize( )
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute( )
-  {}
+  {

```

Comment:
This would be a little cleaner to have a left and right variable set using (condition) ? ifTrue : ifFalse.

Then pass the left and right variables into the TankDriveVolts(Left, right);